ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal

```diff
@@ -150,6 +150,7 @@ library
     crypton,
     deepseq,
     formatting >=6.3 && <7.3,
+    kes-agent,
```

Comment:
This dependency cannot work as is, because it builds `socket-unix` which won't fly on Windows.